### PR TITLE
Color all wires green to save planet(s)

### PR DESCRIPTION
**Pull request by Slava0135**
## Media

image

## Breaking changes

**Changelog**

:cl:
- tweak: As part of our new ecology campaign and green energy investments we colored all wires on station green! Go Green and Save Green!

### CODE REVIEW

File: Resources/Prototypes/Entities/Structures/Power/cables.yml [hunk 1]
```diff
@@ -51,10 +51,12 @@
   description: An orange high voltage power cable.
```
Comment (lzk228) [inline Resources/Prototypes/Entities/Structures/Power/cables.yml:51]: orang

File: Resources/Prototypes/Entities/Structures/Power/cables.yml [hunk 2]
```diff
@@ -48,13 +48,15 @@
   parent: CableBase
   id: CableHV
   name: HV power cable
-  description: An orange high voltage power cable.
+  description: An high voltage power cable.
```
Comment (lzk228) [inline Resources/Prototypes/Entities/Structures/Power/cables.yml:51]: A* 🤓

**Comment by Hmeister-real** [comment]
wait why is this merged its not april fools yet, what the hell

**Comment by mirrorcult** [comment]
image

**Comment by Hmeister-real** [comment]
ohh right

**Pull request merged**